airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -71,6 +72,14 @@ public AirbyteConnectionStatus check(JsonNode config) {
       // attempt to get metadata from the database as a cheap way of seeing if we can connect.
       database.bufferedResultSetQuery(conn -> conn.getMetaData().getCatalogs(), JdbcUtils::rowToJson);
 
+      // verify we have write permissions on the target schema by creating a table with a random name,
+      // then dropping that table
+      String outputSchema = namingResolver.getIdentifier(config.get("schema").asText());
+      String outputTableName = "_airbyte_connection_test_" + UUID.randomUUID().toString().replaceAll("-", "");
+      sqlOperations.createSchemaIfNotExists(database, outputSchema);
+      sqlOperations.createTableIfNotExists(database, outputSchema, outputTableName);
+      sqlOperations.dropTableIfExists(database, outputSchema, outputTableName);
```

Comment:
don't we need to drop the schema too?

Reply:
We wouldn't wnat to drop the schema if it already existed

Reply:
ignore this. i thought this was a test schema.